examples/hybrid/src/main.rs

```diff
@@ -0,0 +1,53 @@
+#![feature(let_chains)]
+
+mod grpc;
+mod web;
```

Comment:
Honestly I think we should just put everything in this file. Splitting things across multiple files just makes the example harder to read. I think most people read examples in the browser which makes it hard to switch between multiples files.